core/templates/css/README.md

```diff
@@ -0,0 +1,20 @@
+Steps to generate oppia-material.css file:
```

Comment:
Give more detail here, before this, about:

- What this file contains
- How it interoperates with the oppia.css file
- When to change this file vs changing oppia.css (what are the "rules" that developers should follow)
- Any other "background" information around the "why" of this file

Take a look at the README in core/templates/expressions for an example of the sort of detail and exposition that is needed here. The explanation should be forward-looking and be aimed at an audience who is reading this after the Angular migration (and who didn't know that we had to do an Angular migration).

Reply:
Oh... this is temporary 😅. This file won't stay after migration.